spark-board-ui/src/App.jsx

```diff
@@ -97,10 +106,8 @@ export default function App() {
                 return node;
             }));
         }
-    }, [settings, nodes, setNodes]);
-
-    // controls the column for which the column graph is shown
-    const [selectedColumn, setSelectedColumn] = useColumnGraphState(null);
+        setSelectedColumn(selectedColumn);
+    }, [settings, nodes, setNodes, setSelectedColumn]);
```

Comment:
I think it should depend on `selectedColumn` and not `setSelectedColumn`

Reply:
Now it depends on the columnGraph, so the dependency on selectedColumn isn't needed

Reply:
Removed that setter